.github/workflows/static-analysis.yml

```diff
@@ -81,8 +81,12 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      - name: Setup Node
```

Comment:
> the `markdownlint-cli2-action` has no option to install custom rules.

I've reported https://github.com/DavidAnson/markdownlint-cli2-action/issues/134 about this.